src/index.html

```diff
@@ -16,17 +16,23 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="input input__line input__line--big">
+      <input
+        class="input__big"
+        type="text"
+        data-qa="keypress big"
+        placeholder="Try “Los Angeles“"
```

Comment:
Keep your attributes correctly formatted. If the HTML-element has long attribute values or the number of attributes is more than 2, start each one, including the first, on a new line with 2-space indentation related to the tag.